R/linters_config.R

```diff
@@ -1,24 +1,22 @@
+gDR_undesirable_functions <-
+  lintr::modify_defaults(
+    defaults = lintr::default_undesirable_functions,
+    "library" = NULL,
+    # we prefer not to use these functions because we use data.table as the primary data format
+    "assert_data_frame",
```

Comment:
consider some hints if the linter fails, e.g.
`"assert_data_frame" = "please use 'assert_data_table' instead (our primary data format)"`

Reply:
Good point.